geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloBackedRasterOperations.scala

```diff
@@ -103,14 +115,74 @@ class AccumuloBackedRasterOperations(val connector: Connector,
     adaptIterator(batchScanner.iterator)
   }
 
+  def getBounds(): BoundingBox = {
+    ensureTableExists(GEOMESA_RASTER_BOUNDS_TABLE)
+    val scanner = connector.createScanner(GEOMESA_RASTER_BOUNDS_TABLE, authorizationsProvider.getAuthorizations)
+    scanner.setRange(new Range(getBoundsRowID))
+    val resultingBounds = scanner.iterator()
+    if (resultingBounds.isEmpty) {
+      BoundingBox(-180, 180, -90, 90)
+    } else {
+      //TODO: add fix for stuff crossing anti-meridian, may not be an issue...
```

Comment:
If you have a second, please add a ticket for this todo and reference it by url.